facia-purger/build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "facia-purger"
 
-version := "1.0"
+version := "1.0.0"
```

Comment:
so it will always be version 1.0.0 but we will actually version it with the build number?